### PR TITLE
Small docs issues

### PR DESCRIPTION
**Pull request by rkuchan**
So far, fixes:
* https://github.com/influxdata/docs.influxdata.com/issues/419 (add minutes as valid duration in `spec.md`)
* https://github.com/influxdata/docs.influxdata.com/issues/306 (add note about floating point math to `functions.md`)
* Part of https://github.com/influxdata/docs.influxdata.com/issues/436 (missing Chronograf in the navbar)
* https://github.com/influxdata/docs.influxdata.com/issues/304 (add documentation for InfluxDB environment variables)

### CODE REVIEW

File: content/influxdb/v0.13/administration/config.md [hunk 1]
```diff
@@ -50,6 +50,35 @@ A new configuration file should be generated every time InfluxDB is upgraded.
 
 See the [installation documentation](/influxdb/v0.13/introduction/installation/#generate-a-configuration-file) for more detail on generating and using configuration files.
 
+## Environment variables
+
+Set any configuration option with an environment variable.
+At the start of the process, InfluxDB checks for environment variables of the
```
Comment (beckettsean) [inline content/influxdb/v0.13/administration/config.md]: maybe reword to "When the process launches, InfluxDB checks..."

**Comment by beckettsean** [comment]
one note, otherwise LGTM

**Comment by rkuchan** [comment]
@kostasb, for https://github.com/influxdata/docs.influxdata.com/issues/306, what functions should include that warning?

**Comment by kostasb** [comment]
@rkuchan 

So far the following functions are proven to be affected by floating point precision so a warning should go with them:

spread
mean
sum
stddev

The epsilon from our tests is estimated at 9 decimals (0.000000001) for 200K floats, but will vary depending on the size and shape of data.

The warning syntax looks good to me.

**Comment by rkuchan** [comment]
Thanks, @beckettsean and @kostasb!
